proto/iceflow.proto

```diff
@@ -43,3 +43,28 @@ message RepartitionResponse {
   uint32 lower_partition_bound = 2;
   uint32 upper_partition_bound = 3;
 }
+
+service NodeExecutor {
+  /// Sends a report about a congestion on an edge within the Iceflow system
+  /// to the NodeExecutor.
+  rpc ReportCongestion(CongestionReportRequest) returns (CongestionReportResponse);
+}
+
+message CongestionReportRequest {
+  /// The specifier of the congested edge.
+  string edge_name = 1;
+
+  /// An identifier of the cause for the congestion.
+  CongestionReason congestion_reason = 2;
+  // TODO: Should the amount of congestion be quantified somehow?
+}
+
+message CongestionReportResponse {
+  // TODO: Does the NodeExecutor need to report something to the Node Instance?
```

Comment:
I don't think so, at least for now.

Reply:
Thanks, I removed the TODO via b1f352e8292ba2c2a1b4e1f2b956c0408aa87379 :)